src/structlog/stdlib.py

```diff
@@ -393,16 +393,22 @@ class ProcessorFormatter(logging.Formatter):
 
     .. versionadded:: 17.1.0
     """
-    def __init__(self, processor, foreign_pre_chain=None, *args, **kwargs):
+    def __init__(self, processor, foreign_pre_chain=None,
+                 unset_record_stack=False, *args, **kwargs):
```

Comment:
How does this look now?

This modified version of record doesn't really go anywhere except the base implementation of `format()`, so maybe `append_stack=True` parameter that does the opposite thing would be better.

What do you think?